docker-compose-local.yml

```diff
@@ -59,10 +59,11 @@ services:
     environment:
       - FTP_IP=127.0.0.1
       - FLAGS=-Y 0 -d -d
+
   pg:
-    image: postgres:14.3-alpine
-    volumes:
-      - ./pg/scripts:/scripts
-      - ./pg/input_data:/input_data
+    profiles: ["gotenzymes"]
```

Comment:
the profile is what makes it easy to exclude this container from standard commands (`build-stack`, `deploy-stack` etc.